src/ducks/apps/components/ApplicationRouting.jsx

```diff
@@ -60,13 +64,37 @@ export class ApplicationRouting extends Component {
                     fetchError={fetchError}
                     app={app}
                     isInstalling={isInstalling}
-                    isFetching={isFetching}
                   />
                 )
               }
             }
           }}
         />
+        <Route
+          path={`/${parent}/:appSlug/configure`}
+          render={({ match }) => {
```

Comment:
To make the code a little bit clearer, do you think it's interesting to externalize the `render` method here ? It fees like a pretty huge block of code to be just in a Component's property.

Reply:
I don't know, these parts seems similar but not enough to be externalized I think